src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_ARRAY_SIZE = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private List<K> keyList;
+    private Node<K, V>[] elements;
+    private int size;
+
+    public MyHashMap() {
+        elements = (Node<K,V>[])new Node[DEFAULT_ARRAY_SIZE];
+        keyList = new ArrayList<>();
+    }
+
     @Override
     public void put(K key, V value) {
-
+        if (size > elements.length * LOAD_FACTOR) {
+            resize();
+        }
+        putValue(key, value, elements);
     }
```

Comment:
make this check in resize using " if ( size + 1 > ... )" and only call resize here